### PR TITLE
tutum 0.16.0

### PR DESCRIPTION
**Pull request by tifayuki**

### CODE REVIEW

File: Library/Formula/tutum.rb [hunk 1]
```diff
@@ -22,8 +22,8 @@ class Tutum < Formula
   end
 
   resource "docker-py" do
-    url "https://pypi.python.org/packages/source/d/docker-py/docker-py-1.2.2.tar.gz"
```
Comment (DomT4) [inline Library/Formula/tutum.rb]: This appears to be a downgrade of versions, from 1.2.2 to 1.1.0. Intentional?

File: Library/Formula/tutum.rb [hunk 2]
```diff
@@ -52,8 +52,8 @@ class Tutum < Formula
   end
 
   resource "requests" do
-    url "https://pypi.python.org/packages/source/r/requests/requests-2.7.0.tar.gz"
-    sha256 "398a3db6d61899d25fd4a06c6ca12051b0ce171d705decd7ed5511517b4bb93d"
+    url "https://pypi.python.org/packages/source/r/requests/requests-2.5.3.tar.gz"
```
Comment (DomT4) [inline Library/Formula/tutum.rb]: Same here. Version downgrade?

**Comment by tifayuki** [comment]
In previous version, we didn't fix the version of dependencies, and it causes troubles some time. 
As a result, we fix the version in the pypi release from this version and make consistence here in the brew release.

**Comment by DomT4** [comment]
The downgrade of `Requests` is kind of a shame because the downgrade reintroduces the [CVE-2015-2296](http://www.cvedetails.com/cve/CVE-2015-2296/) vulnerability ([Openwall Report](http://www.openwall.com/lists/oss-security/2015/03/14/4)).

I think @tdsmith decided not to bump all the `requests` resources in Homebrew at the time, but I'm semi-reluctant to actively reintroduce a CVE.

Will go with whatever Tim says here.

**Comment by mikemcquaid** [comment]
Yeh, :-1: on reintroducing a CVE. Let's leave the versions as-is unless it's required to build or fix issues.

**Comment by tifayuki** [comment]
I will update the dependency to a new version of docker-py and requests and create a new PR

**Comment by mikemcquaid** [comment]
@tifayuki If you create PRs from a non-master branch you can just push (or force-push) more commits there and update it.

**Comment by tifayuki** [comment]
I created the PR from a master branch. So should I close this PR and create a new one ? @mikemcquaid

**Comment by mikemcquaid** [comment]
I think you can just make another commit and push it to your `master` branch. If that doesn't work: create a new PR, thanks.

**Comment by tifayuki** [comment]
@mikemcquaid I have upgraded request, can you merge it ? thank you

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**